jetty-server/src/main/java/org/eclipse/jetty/server/HttpOutput.java

```diff
@@ -627,6 +626,7 @@ public void close() throws IOException
                 catch (Throwable t)
                 {
                     onWriteComplete(true, t);
+                    throw t;
```

Comment:
I'm not sure we want to throw here... at least needs a comment. `onWriteComplete()` takes the throwable, and we are closing the output, so no more writes are possible, etc.
Unless this is the only "write", we won't be able to send another response by rethrowing, and most of the exception handling of `close()` is just catch and ignore.

Reply:
Because we throw from write in the same circumstance.  We need to be consistent

Reply:
Okay, I see also `flush()` rethrows.